### DISCUSSION TITLE
SLC #2 deployment

### DISCUSSION THREAD

**Discussion by g105b** [SuperHyperInstantFutureTime/TrackShift#109] [Work log]
Today we're developing together in person, and I have some work to complete on the Audit Trail side of things. Once I'm happy with it, we can deploy to live as our second production deployment.

We'll log anything that happens today here to keep up with the ongoing documentation.

**Comment by g105b** [comment]
@richardbirkin we've got our SLC2 code live on https://slc2.dev.trackshift.app/ for testing.

Please can you run some manual tests through the functionality, and if you're happy, we can release SLC2 to production later today.

**Comment by richardbirkin** [comment]
Did you merge `slc2-biff-styles` and `slc2-audit-biff-styles` ??

**Reply by richardbirkin** [reply]
@g105b

**Reply by g105b** [reply]
Nope. Doing another deployment now. Give me 5 mins.

**Comment by g105b** [comment]
By the way, @richardbirkin , the issue with the escape key was a simple one to fix. "Escape" is already bound to a function in browsers - cancel navigation. That's why every key other than escape was working. So I fixed it by preventing the default function on the escape key. The code you wrote last week was fine, it just needed this extra line.

**Comment by richardbirkin** [comment]
Cool!

There are still some design and usability tweaks to do but they will also apply to SLC3-6 so can be done in due course. 

I'm signing off SLC2. 

Onto SLC3!
